README.md

```diff
@@ -23,9 +25,11 @@ DATABASE_NAME     = steampulse
 DATABASE_USERNAME = steampulse_admin
 DATABASE_PASSWORD = XXXXXXXXXXX
 DATABASE_ENDPOINT = XXXXXXXXXXX
+DASHBOARD_URL     = XXXXXXXXXXX
+EMAIL_SENDER      = XXXXXXXXXXX
 ```
 
-If hosting this service on AWS, you'll need to create your RDS to get the database endpoint.
+If hosting this service on AWS, you'll need to create your RDS to get the database endpoint and dashboard url. Email sender will be your email address which you will need to verify using SES to use the functions.
```

Comment:
```suggestion
If hosting this service on AWS, you'll need to create your RDS to get the database endpoint as an output from terraform and get the dashboard url from the ECS service on AWS, since terraform can't output this. Email sender will be your email address which you will need to verify using SES to use the functions.
```